pkg/storages/storage/folder.go

```diff
@@ -27,6 +27,8 @@ type Folder interface {
 	ReadObject(objectRelativePath string) (io.ReadCloser, error)
 
 	PutObject(name string, content io.Reader) error
+	
+	CopyObject(baseBackupPath string, objectRelativePath string, dstObject string) error
```

Comment:
Is it possible to leave only the `fromPath` and `toPath`? Like this: `CopyObject(fromPath, toPath string) error`. It would be more universal.